di_website/publications/mixins.py

```diff
@@ -89,11 +89,11 @@ class Meta:
 
     @cached_property
     def publication_date(self):
-        if self.published_date:
+        if hasattr(self, 'published_date') and self.published_date:
```

Comment:
This page will always have a `published_date` property, so this isn't required. This error was originating from the parent on line 96. Still, not harming anyone.